controllers/terraform/logging.go

```diff
@@ -43,6 +43,6 @@ func getPodLog(ctx context.Context, client kubernetes.Interface, namespace, jobN
 		return "", err
 	}
 	logContent := buf.String()
-	klog.Info("pod logs", "Pod", pod.Name, "Logs", logContent)
+	klog.V(4).Info("pod logs", "Pod", pod.Name, "Logs", logContent)
```

Comment:
Why is the changed line better?

Reply:
Level 2 is usually used prod, which can effectively avoid too much output of container logs

Reply:
Cool. Can you share a link on the klog level. I didn't find any useful references.

Reply:
Ref：https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md

Reply:
Great! Very useful link ever!